### DISCUSSION TITLE
Unable to connect with the app using 2 factor authentification

### DISCUSSION THREAD

**Discussion by TheVgam3r** [bttv-android/bttv#594] [General]
i tried to connect with my account that i've setup 2 factor authentification on it a few minutes ago cause i've changed my password before that and the BTTV app wont let me connect to my account and enter the 2FA code. BUT, on the original Twitch app, it works perfectly fine...

**Comment by FoseFx** [comment]
I'm just going to assume this is related to this issue: https://github.com/bttv-android/bttv/issues/588

The passport backend started blocking bttv-android requests, will be fixed in a day or two

**Reply by TheVgam3r** [reply]
cool thank you !
